<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageProcessClass</class>
 <widget class="QMainWindow" name="ImageProcessClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImageProcess</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="imageLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>471</width>
      <height>291</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>600</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="hue">
     <attribute name="title">
      <string>Hue</string>
     </attribute>
     <widget class="QSlider" name="hueSlider">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>8</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>360</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="hueLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>13</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>Input:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="hueButton">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>8</y>
        <width>75</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Adjust</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>10</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0°</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>10</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>360°</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="hueInput">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>9</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maximum">
       <double>360.000000000000000</double>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="saturation">
     <attribute name="title">
      <string>Saturation</string>
     </attribute>
     <widget class="QLabel" name="satLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>13</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>Input:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="satButton">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>8</y>
        <width>75</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Adjust</string>
      </property>
     </widget>
     <widget class="QSlider" name="satSlider">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>8</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-100</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>325</x>
        <y>9</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>-100</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>9</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>100</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="satInput">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>9</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimum">
       <double>-100.000000000000000</double>
      </property>
      <property name="maximum">
       <double>100.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="intensity">
     <attribute name="title">
      <string>Intensity</string>
     </attribute>
     <widget class="QSlider" name="intSlider">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>8</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-100</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="intButton">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>8</y>
        <width>75</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Adjust</string>
      </property>
     </widget>
     <widget class="QLabel" name="intLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>13</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>Input:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>325</x>
        <y>9</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>-100</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>9</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>100</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="intInput">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>9</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimum">
       <double>-100.000000000000000</double>
      </property>
      <property name="maximum">
       <double>100.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionReverse"/>
    <addaction name="separator"/>
    <addaction name="actionRotate_CW"/>
    <addaction name="actionRotate_CCW"/>
    <addaction name="separator"/>
    <addaction name="actionChoose_Clip_Area"/>
    <addaction name="actionExit_Choose"/>
    <addaction name="actionClip"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open exist file</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save the file</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit the program</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save as..</string>
   </property>
   <property name="toolTip">
    <string>Save the file as...</string>
   </property>
  </action>
  <action name="actionReverse">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Reverse</string>
   </property>
   <property name="toolTip">
    <string>Reverse the image</string>
   </property>
  </action>
  <action name="actionRotate_CW">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Rotate CW</string>
   </property>
   <property name="toolTip">
    <string>Rotate 90 degree clockwise</string>
   </property>
  </action>
  <action name="actionRotate_CCW">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Rotate CCW</string>
   </property>
   <property name="toolTip">
    <string>Rotate 90 degree counter clockwise</string>
   </property>
  </action>
  <action name="actionChoose_Clip_Area">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Choose Clip Area</string>
   </property>
   <property name="toolTip">
    <string>Choose the clip area</string>
   </property>
  </action>
  <action name="actionClip">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Clip</string>
   </property>
   <property name="toolTip">
    <string>Clip the image with the chosen area</string>
   </property>
  </action>
  <action name="actionExit_Choose">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Exit Choose</string>
   </property>
   <property name="toolTip">
    <string>Exit choosing clip area mode</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="imageprocess.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>ImageProcessClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
